te_dp/src/te_dp.c

```diff
@@ -175,9 +175,6 @@ void load_random_session_data(te_session_t * session)
 void send_session_requests(te_session_t *session, unsigned int num_requests)
 {
     unsigned int iter = 0;
-    if (unlikely(is_session_config_state_stopped(session->session_cfg_p))) {
```

Comment:
We anyways don't support pause and resume APIs. Removing unnecessary if check, which would always fail.